.ci/Jenkinsfile

```diff
@@ -316,8 +316,15 @@ def generateFunctionalTestStep(Map args = [:]){
   }
   tags += excludeNightlyTag
 
+  workerLabels = "${platform}"
+  if (platform != "arm") {
```

Comment:
can you describe the intent of the branching?  I see it has to do with immutable workers, but don't have context - maybe that is what you could comment better on?

Reply:
Exactly, this is how Infra's workers are labelled: arm are not immutable yet